build/lib/versions.txt

```diff
@@ -8,13 +8,13 @@ bcpkix-jdk15on.jar                  | 1.53
 bcprov-jdk15on.jar                  | 1.53                                                         | See http://www.bouncycastle.org/licence.html
 cglib.jar                           | 2.1.3 (JMock 2.1.0)                                          |
 commons-lang.jar                    | 2.3                                                          | Apache 2.0
-commons-logging.jar                 | Jetty 5.1.10                                                 | Apache 2.0
```

Comment:
I think that the `CommonsLogFactory` class still depends on this library. If we remove this, will we break the build?